src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -1,37 +1,37 @@
 package cromwell.webservice
 
-import akka.actor.{Actor, ActorRef, Props}
+import java.util.UUID
+import akka.actor.{ActorRef, Actor, Props}
 import akka.pattern.ask
 import akka.util.Timeout
 import cromwell.binding.{WdlSource, WorkflowRawInputs}
-import cromwell.engine
+import cromwell.{binding, engine}
 import cromwell.engine._
 import cromwell.parser.WdlParser.SyntaxError
 import cromwell.webservice.CromwellApiHandler._
 import cromwell.webservice.PerRequest.RequestComplete
 import spray.http.StatusCodes
-
 import scala.concurrent.duration._
-import scala.util.{Failure, Success}
+import scala.util.{Try, Failure, Success}
 
 object CromwellApiHandler {
   sealed trait WorkflowManagerMessage
   case class SubmitWorkflow(wdl: WdlSource, inputs: WorkflowRawInputs) extends WorkflowManagerMessage
   case class WorkflowStatus(id: WorkflowId) extends WorkflowManagerMessage
+  case class WorkflowOutputs(id: WorkflowId) extends WorkflowManagerMessage
 
   def props(workflowManagerActorRef : ActorRef): Props = {
     Props(new CromwellApiHandler(workflowManagerActorRef))
   }
 }
 
-class CromwellApiHandler(workflowManagerActorRef : ActorRef) extends Actor {
+class CromwellApiHandler(workflowManager : ActorRef) extends Actor {
   import context.dispatcher
+  implicit val timeout = Timeout(2.seconds)
 
   override def receive = {
-
     case WorkflowStatus(id) =>
-      implicit val timeout = Timeout(2.seconds)
-      val workflowManagerResponseFuture = ask(workflowManagerActorRef, WorkflowManagerActor.WorkflowStatus(id)).mapTo[Option[WorkflowState]]
+      val workflowManagerResponseFuture = ask(workflowManager, WorkflowManagerActor.WorkflowStatus(id)).mapTo[Option[WorkflowState]]
```

Comment:
why not regular question mark syntax here with inlined `onComplete` handler here and below?